### PR TITLE
Feature#14/check validation

### LINKED ISSUE #14: Validate that an employee exists and show a confirmation message
# REQUIREMENTS
It must validate the information provided by the employee and show a confirmation message.
- Check if the input contains the needed information
- Search in database the information to validate
- Show a message after validation.
# DESIGN

image

### PR DESCRIPTION
**Pull request by nallely64**
# Description
 - What?:  Implementation of employee validation on check in and display of an alert message of succes or failure, and model´s relations.
 - Why?:  To inform to the employee if his check in was succesful or not.
 - For more information, visit #14 issue.

  ## Screenshots
Failure check in:
image
Succesful check in:
image (1)

### CODE REVIEW

File: app/controllers/home_controller.rb [hunk 1]
```diff
@@ -1,4 +1,21 @@
 class HomeController < ApplicationController
+  #Si existe el numero privado del usuario
+  #Registrar fecha de registro y mostrar mensaje de confirmacion
+  #Sino existe, mostrar mensaje de error
   def index
+    @assistance = Assistence.find_by(id: 1)
```
Comment (YilKanda) [inline app/controllers/home_controller.rb]: Why do you only need the assistance of id 1?

Comment (germanruzca) [reply app/controllers/home_controller.rb]: It was just a test line, already removed :)

File: app/views/home/index.html.erb [hunk 2]
```diff
@@ -1,19 +1,23 @@
 <div class="main">
   <div class="time">
     <span class="date">
-      10 marzo 2022
+      <%= I18n.l(Date.today, format: '%A, %d de %B de %Y') %>
```
Comment (JuanequeX) [inline app/views/home/index.html.erb:4]: Nice

**Review by JuanequeX — APPROVED** [review]
LGTM

**Review by YilKanda — APPROVED** [review]
LGTM!

**Review by xFerchoVJ — APPROVED** [review]
LGTM

**Pull request merged**